R/p_test.R

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#' @title Calculate the p-value of the null hypothesis that two outcomes are from the same dataset
+#'
+#' @description
+#' Specify an outcome variable and return p-test outputs.
+#' All numeric variables in the dataset are used as predictor variables.
+#'
+#' @param data A Person Query dataset in the form of a data frame.
+#' @param outcome A string specifying a binary variable, i.e. can only contain
+#' the values 1 or 0. Used to group the two distributions.
+#' @param behavior A character vector specifying the column to be used as the behavior to test.
+#'
+#' @import dplyr
+#'
```

Comment:
@m-m-powers Might be good to add an example of what this could look like if you run with `sq_data`? 

We can add a roxygen tag `@examples` and code underneath it. I think it's fine not to add the `dontrun{}` braces so that R can tell us in its automated checks in case the function breaks.